tomviz/AddPythonTransformReaction.cxx

```diff
@@ -358,7 +358,7 @@ OperatorPython* AddPythonTransformReaction::addExpression(DataSource* source)
       addPythonOperator(source, this->scriptLabel, this->scriptSource,
                         arguments);
     }
-  } else if (scriptLabel == "Reconstruct (Constraint-based Direct Fourier)") {
+  }/* else if (scriptLabel == "Reconstruct (Constraint-based Direct Fourier)") {
```

Comment:
Why not just remove these lines?

Reply:
I think this is one of the reasons this commit was still work in progress before I cherry-picked it...  updated.